From: Amir Vadai <amirv@mellanox.com>
Date: Tue, 25 Feb 2014 18:17:51 +0200
Subject: net/mlx4_en: Expose port number through sysfs
Patch-mainline: v3.15-rc1
Git-commit: 76a066f2a2a0268b565459c417b59724b5a3197b
References: bug#919382 FATE#317529

Initialize dev_port with port number (0 based) to be accessed through
sysfs from user space.

Signed-off-by: Amir Vadai <amirv@mellanox.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Benjamin Poirier <bpoirier@suse.de>
---
 drivers/net/ethernet/mellanox/mlx4/en_netdev.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/net/ethernet/mellanox/mlx4/en_netdev.c
+++ b/drivers/net/ethernet/mellanox/mlx4/en_netdev.c
@@ -2043,6 +2043,7 @@ int mlx4_en_init_netdev(struct mlx4_en_d
 
 	SET_NETDEV_DEV(dev, &mdev->dev->pdev->dev);
 	dev->dev_id =  port - 1;
+	dev->dev_port = port - 1;
 
 	/*
 	 * Initialize driver private data
